website/content/api-docs/system/internal-counters.mdx

```diff
@@ -89,24 +89,180 @@ $ curl \
 
 ## Client Count
 
-This endpoint returns the number of clients per namespace, as the sum of clients calculated from the use 
-of active entities and non-entity tokens.
-An "active entity" is a distinct entity that has created one or more tokens in the given time period.
-A "non-entity token" is a token with no attached entity ID. 
-Both non-entity tokens and active entities have distinct client IDs. For more information on how clients
-map to these client IDs, and how clients are counted, please visit the 
-[client count](/docs/concepts/client-count) concepts page.
-
-A time period may be specified; otherwise it reports on a default reporting period, such as the
-previous twelve calendar months. Reports are only available with month granularity, after each month
-has completed. The response includes the actual time period covered, which may not exactly match
-the query parameters due to the monthly granularity of the data, or missing months in the requested
-time range.
-
-The response will include all child namespaces of the namespace in which the request was made.
-If some namespace has subsequently been deleted, its path will be listed as "deleted namespace :ID:".
-Deleted namespaces are reported only for queries in the root namespace, because the information about
-the namespace path is unknown.
+This endpoint returns client activity information for a given billing
+period. The billing period is represented by the `start_time` and `end_time`
+parameters.
+
+There are a few things to be known while using this API.
+
+- The activity information only accounts for activity that has happened for the
+latest contiguous months in the billing period. For example, if the billing
+period is from Jan 2022 to June 2022, and say the activity subsystem in Vault
+was not capturing data for the months Jan to March for any reason, the response
+will only include information for the months of May and June.
+
+- The response contains the total activity for the billing period and the
+attributions of the total activity against specific components in Vault. This
+helps in understanding the total activity better by knowing which components in
+Vault are leading to that top level activity count. First level of attribution
+breakdowns are by namespaces and months, under the `by_namespaces` and `months`
+JSON block respectively.
+
+```json
+{
+   "by_namespace":[],
+   "months":[],
+}
+```
+
+- `by_namespaces` breakdowns provide attributions of each namespace to the total
+activity count. These namespace attributions are further broken down by mount
```

Comment:
Suggest to rewrite as:
Mount level attributions further break down these namespaces attributions, wherein information can be found on the attributions of each mount path within a given namespace to the overall activity of the namespace.

Reply:
Thanks. Done!